logstage/logstage-sink-file/src/main/scala/com/github/pshirshov/izumi/models/LogFile.scala

```diff
@@ -0,0 +1,121 @@
+package com.github.pshirshov.izumi.models
+
+import java.nio.file.{FileAlreadyExistsException, Files, Paths, StandardOpenOption}
+import scala.io.Source
+import scala.util.{Failure, Success, Try}
+import LogFile.TryOps._
+
+trait LogFile {
+  def name: FileId
+
+  def path: String
+
+  def status: LogFileStatus
+
+  def write(item: String): Try[Unit]
```

Comment:
```Try``` is too specific and prevents dummy implementation. Try is not required for safety here, as you can put a top-level try in ```.flush()``` and implement the rest without using a monad. You can't abstract over monads (provide dummy implementation for ```Id```) without importing ```cats.Monad```, so better not use them at all. Instead consider bundling the necessary operations into ```LogFile```